controllers/pullRequestsController.js

```diff
@@ -0,0 +1,58 @@
+const logger = require('../utils/logger')
+const githubService = require('../services/githubService')
+
+/**
+ * Loops over an array of objects, takes a value corresponding to key provided and saves it in an array
+ *
+ * @param arrayOfObjects {Array} - Array of objects to loop over
+ * @param key {String} - Value corresponding to this key is saved
+ */
+
+const getNames = (arrayOfObjects, key) => {
+  const names = []
+  arrayOfObjects.forEach((object) => {
+    names.push(object[key])
+  })
+  return names
+}
+
+/**
+ * Collects all pull requests and sends only required data for each pull request
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+
+const getPRdetails = async (req, res) => {
+  try {
+    const data = await githubService.fetchPRsByUser(req.params.id)
+
+    if (data.total_count) {
+      const allPRs = []
+      data.items.forEach(({ title, html_url: htmlUrl, state, created_at: createdAt, updated_at: updatedAt, draft, labels, assignees }) => {
+        const allAssignees = getNames(assignees, 'login')
+        const allLabels = getNames(labels, 'name')
+        allPRs.push({
+          title: title,
+          url: htmlUrl,
+          state: state,
+          created_at: createdAt,
+          updated_at: updatedAt,
+          ready_for_review: state === 'closed' ? false : !draft,
```

Comment:
`snake_case -> camelCase -> snake_case` conversion detected.
Any particular reason?

Reply:
No. Fixed it.